ui_framework/src/components/toast/_toast.scss

```diff
@@ -2,20 +2,27 @@
   @include kuiBottomShadow;
 
   position: relative;
-  padding: $kuiSizeM;
+  padding: $kuiSize;
   background-color: $kuiColorEmptyShade;
   border: $kuiBorderThin;
+  width: $kuiToastWidth;
```

Comment:
Bordering on being dogmatic, but it seems like we should be setting this on `kuiGlobalToastListItem` instead, since that's the container that's concerned with size and layout.